assets/js/components/Dashboard/Dashboard.tsx

```diff
@@ -91,7 +91,8 @@ const Dashboard = (props: { page: string }): JSX.Element => {
       // This catches on Silver Line, which we haven't really discussed how it should be treated.
       // Right now, the places list for SL is empty because its screens are all getting listed as buses.
       // It should probably treated as a bus route, but still a question for Adam.
-      modeLineFilterValue.label.includes("Line")
+      modeLineFilterValue.label.includes("Line") ||
+      modeLineFilterValue.label === "Mattapan"
```

Comment:
I added that comment there, before I knew Adam wanted Silver Line to be treated like the other lines. Not your responsibility, but mind removing the comments now?